packages/api/lib/writeRecords/write-granules.js

```diff
@@ -390,197 +285,242 @@ const _generateFilesFromGranule = async ({
- * @param {Object} params.granule - An API granule object
- * @param {Object} params.provider - Provider object
- * @param {string} params.workflowStatus - Workflow status
- * @param {string} params.collectionCumulusId
- *   Cumulus ID for collection referenced in workflow message, if any
- * @param {string} params.executionCumulusId
- *   Cumulus ID for execution referenced in workflow message, if any
- * @param {Knex} params.knex - Client to interact with PostgreSQL database
- * @param {Object} [params.error] - Workflow error, if any
- * @param {string} [params.executionUrl]
- *   Step Function execution URL for the workflow, if any
- * @param {Object} [params.processingTimeInfo]
- *   Processing time information for the granule, if any
- * @param {string} [params.providerCumulusId]
- *   Cumulus ID for provider referenced in workflow message, if any
- * @param {string} [params.pdrCumulusId]
- *   Cumulus ID for PDR referenced in workflow message, if any
- * @param {Object} [params.granuleModel]
- *   Optional override for the granule model writing to DynamoDB
- *
- * @returns {Promise}
- * @throws
+ * Write a granule record to DynamoDB and PostgreSQL
+ * param {PostgresGranule} postgresGranuleRecord,
+ * param {DynamoDBGranule} dynamoGranuleRecord,
+ * param {number} executionCumulusId,
+ * param {Knex} knex,
+ * param {Object} granuleModel = new Granule(),
+ * returns {Promise}
+ * throws
  */
 const _writeGranule = async ({
-  collectionId,
-  granule,
-  pdrName,
-  provider,
-  workflowStartTime,
-  workflowStatus,
-  queryFields,
-  collectionCumulusId,
+  postgresGranuleRecord,
+  dynamoGranuleRecord,
   executionCumulusId,
   knex,
-  error,
-  executionUrl,
-  processingTimeInfo,
-  providerCumulusId,
-  pdrCumulusId,
   granuleModel,
-  updatedAt = Date.now(),
+  granulePgModel,
   esClient,
 }) => {
   let granuleCumulusId;
-  const files = await _generateFilesFromGranule({ granule, provider });
 
   await knex.transaction(async (trx) => {
-    granuleCumulusId = await _writeGranuleViaTransaction({
-      granule,
-      processingTimeInfo,
-      error,
-      provider,
-      workflowStartTime,
-      workflowStatus,
-      queryFields,
-      collectionCumulusId,
-      providerCumulusId,
+    granuleCumulusId = await _writePostgresGranuleViaTransaction({
+      granuleRecord: postgresGranuleRecord,
       executionCumulusId,
-      pdrCumulusId,
       trx,
-      updatedAt,
-      files,
+      granulePgModel,
     });
-
-    return writeGranuleToDynamoAndEs({
-      granule,
-      executionUrl,
-      collectionId,
-      provider,
-      workflowStartTime,
-      error,
-      pdrName,
-      workflowStatus,
-      processingTimeInfo,
-      queryFields,
-      granuleModel,
+    await writeGranuleToDynamoAndEs({
+      dynamoGranuleRecord,
       esClient,
-      updatedAt,
-      files,
+      granuleModel,
     });
+    // return granuleModel.storeGranule(dynamoGranuleRecord);
```

Comment:
I believe this should be uncommented 
```suggestion
     return granuleModel.storeGranule(dynamoGranuleRecord);
```

Reply:
Looking into this further, it looks like this can be removed altogether because of the storing of the granule occurs in `writeGranuleToDynamoAndEs`